framework/src/modules/chain/chain.js

```diff
@@ -428,21 +421,13 @@ module.exports = class Chain {
 					'Sync time triggered'
 				);
 				if (!this.loader.syncing() && this.blocks.isStale()) {
-					this.scope.sequence.add(
-						sequenceCB => {
-							this.loader
-								.sync()
-								.then(() => sequenceCB())
-								.catch(err => sequenceCB(err));
-						},
-						syncError => {
-							if (syncError) {
-								this.logger.error('Sync timer', syncError);
-								return cb(syncError);
-							}
-							return cb();
+					await this.scope.sequence.add(async () => {
```

Comment:
i think we also need to return await from here

Reply:
In the jobQueue, the result of the promise is not use and it just need to wait for the execution.
Why do you think it needs to return promise here?

Reply:
because multiple nextSync jobs may be executed if we don't return promise from here. Come to think of it, it will not cause any problems because the preceding `if` statement will return false if syncing process is executing.